src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,46 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import './DropdownMenu.scss';
+import { Person } from '../../types/Person';
+
+type Props = {
+  people: Person[];
+  selectedPerson: Person | null;
+  onSelect: (person: Person) => void;
+};
+
+export const DropdownMenu: FC<Props> = ({
+  people,
+  selectedPerson,
+  onSelect,
+}) => {
+  return (
+    <div className="dropdown-menu" role="menu" data-cy="suggestions-list">
+      <div className="dropdown-content">
+        {people.map(person => {
+          const { name, sex } = person;
+
+          return (
+            <div
+              key={person.slug}
```

Comment:
Key generation on render is not recommended. This is because keys should be stable between multiple renders for performance reasons. You can read more about it [here](https://medium.com/blackrock-engineering/5-common-mistakes-with-keys-in-react-b86e82020052).